cmd/flakechecker/main.go

```diff
@@ -0,0 +1,157 @@
+package main
+
+import (
+	_ "embed"
+	"flag"
+	"fmt"
+	"github.com/carlmjohnson/versioninfo"
+	junit "github.com/joshdk/go-junit"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"github.com/stackrox/junit2jira/pkg/testcase"
+	"os"
+)
+
+const totalRunsLimit = 30
```

Comment:
```suggestion
const minHistoricalRuns = 30
```
I think this expresses the purpose better.